### ISSUE TITLE
Can't load AMD modules that define() without a name

### ISSUE THREAD

**Issue by macu** [ember-cli/ember-cli#2077]
I can't find any way to access some modules while using Bower and app.import in the Brocfile. [By some accounts](http://addyosmani.com/writing-modular-js/) the name and dependencies of the `define` call are optional.

``` JavaScript
// In alertify.js
// AMD and window support
if ( typeof define === 'function' ) {
    define( [], function () {
        return alertify;
    } );
} else if ( !window.alertify ) {
    window.alertify = alertify;
}
```

In the example above, the name is omitted in the `define` call, and I can neither access alertify as a global, nor import it by declaring the exports hash on `app.import`.

Intending to access alertify as a global yields "Uncaught ReferenceError: alertify is not defined"...

``` JavaScript
// In Brocfile
app.import('bower_components/alertify-js/build/alertify.js');
```

Intending to import alertify yields "Uncaught Error: Could not find module alertify" because the name is omitted from `define`.

Is there a way to handle these anonymous define calls, since the name is provided in the exports hash in `app.import`? If I simply add the name to the define call in relevant third-party libraries, then I can import the modules like so:

``` JavaScript
module alertify from 'alertify';
```

**Comment by sirianni** [comment]
Is there a workaround for this?

**Comment by ef4** [comment]
See https://github.com/ember-cli/ember-cli/pull/5512

**Comment by Kerrick** [comment]
For future readers coming from Google, this was eventually implemented and supported in #5976. It is now documented under [Managing Dependencies](https://ember-cli.com/user-guide/#managing-dependencies) in the guides, and looks like this:

```js
app.import('bower_components/ic-ajax/dist/amd/main.js', {
  using: [
    { transformation: 'amd', as: 'ic-ajax' }
  ]
});
```

**Comment by taras** [comment]
FYI PR #5976 was released in 2.9.0
